api/src/main/resources/workbench.yaml

```diff
@@ -2789,6 +2811,33 @@ definitions:
         items:
           $ref: "#/definitions/CohortReview"
 
+  WorkbenchListBillingAccountsResponse:
+    type: object
+    required:
+      - billingAccounts
+    properties:
+      billingAccounts:
+        type: "array"
+        items:
+          $ref: "#/definitions/BillingAccount"
+
+  BillingAccount:
+    type: object
+    required:
+      - isFreeTier
+      - name
+      - displayName
+      - isOpen
+    properties:
+      isFreeTier:
+        type: boolean
+      name:
+        type: string
+      displayName:
+        type: string
+      isOpen:
```

Comment:
Could you add a comment about what it means for a Billing Account to be "open" ?